### ISSUE TITLE
Check for Python module *jinja2*

### ISSUE THREAD

**Issue by paulmenzel** [seqan/seqan#2366]
The configure script should check, if the Python module *jinja2* is installed. Otherwise, the test *build_dox* fails.

```
        Start 379: build_dox
379/434 Test #379: build_dox .........................................................................***Failed    0.07 sec
Traceback (most recent call last):
  File "/dev/shm/seqan/dox/../util/bin/dox.py", line 14, in <module>
    sys.exit(main())
  File "/dev/shm/seqan/dox/../util/bin/dox.py", line 10, in main
    import seqan.dox.pure
  File "/dev/shm/seqan/util/py_lib/seqan/dox/pure.py", line 18, in <module>
    import write_html
  File "/dev/shm/seqan/util/py_lib/seqan/dox/write_html.py", line 13, in <module>
    import jinja2
ImportError: No module named jinja2
```

**Comment by smehringer** [comment]
HI @paulmenzel,

thanks for reporting this. @eseiler can you look into this?

**Comment by rrahn** [comment]
I am not sure we should do this. We don't want our build system to install dependencies into the system. So the recommended way is to create a virtualenv and install the dependencies. Then trigger the build. But this needs to be done externally.  So instead I suggest to add a proper error message by the build system if jinja2 could not be found.

**Comment by rrahn** [comment]
Oh sorry, i was confused about what was expected. So all good from my side.

**Comment by rrahn** [comment]
this was fixed by #2367
